package.json

```
@@ -119,7 +119,7 @@
     "semantic-release": "^21.0.0",
     "ts-node": "^10.9.1",
     "ttypescript": "^1.5.15",
```

```suggestion
    "ttsc": "0.0.5",
```
I think changing this would work nicely.

---

These libraries support TS@5 and use the
and provides the same functionality as ttypescript.